services/mantle/resources/game.py

```diff
@@ -107,8 +107,17 @@ def post(cls):
             )
             success = offside_response.success
         elif event_type == "end":
-            # TODO - handle this
-            pass
+            time_ = int(event.get("time"))
```

Comment:
Don't send time. Read messenger discussion, the `endtime` field should have `DEFAULT NOW()`